tools/msgfplus/msgfplus.xml

```diff
@@ -56,6 +55,13 @@
             -maxCharge $advanced.maxCharge
             -n $advanced.n
             -addFeatures $advanced.addFeatures
+	#if $tsvcheck
+	    &&
+	    msgf_plus
+	        edu.ucsd.msjava.ui.MzIDToTsv
+	        -i "$output_name"
```

Comment:
Ok got it and makes sense! Bash always confuses me though, not using it enough so I forget the details. (the python part was a joke)